tests/integration/test_db_admin.py

```diff
@@ -38,6 +38,7 @@
 
 
 @pytest.mark.group(1)
+@pytest.mark.unstable
```

Comment:
Switchover fails consistently. Created [DPE-6320](https://warthogs.atlassian.net/browse/DPE-6320) to investigate and fix.